### PR TITLE
Update `Project Loved` articles

### PR DESCRIPTION
**Pull request by Hiviexd**
- guideline rewords that were discussed a while ago
- added more info to the team page
- removed detail about colored names because it's incorrect, user colors are not set to project loved team members

### CODE REVIEW

**Review by aceticke — APPROVED** [review]
yeah

File: wiki/People/Project_Loved_Team/en.md [hunk 1]
```diff
@@ -6,14 +6,29 @@ tags:
...
+
+- *Content moderators*, who make sure that every nominated beatmap abides by the [Song Content Rules](/wiki/Rules/Song_Content_Rules) and [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).
+- *Description reviewers*, who check the descriptions written by the captains for spelling and grammar mistakes
+- *Metadata reviewers*, who ensure that the metadata of nominated beatmaps are correct
 
 The [Project Loved group page](https://osu.ppy.sh/groups/31) lists all of the team members. A more detailed member listing for all of the various roles within the Project Loved team, as well as previous members, can be found on the [Project Loved website](https://loved.sh/team).
 
+## Joining the Project Loved Team
+
+Most new team members are selected through an application process held for specific roles in the Project Loved Team to fill empty positions or to assist with workload. The respective application forms are announced in the Project Loved [news posts](https://osu.ppy.sh/home/news), and successful applicants are announced to the public in the subsequent month's post.
```
Comment (Walavouchey) [inline wiki/People/Project_Loved_Team/en.md:23]:
```suggestion
Most new team members are selected through an application process held for specific roles in the Project Loved Team to fill empty positions or to assist with workload. Application forms are announced in Project Loved [news posts](https://osu.ppy.sh/home/news), and successful applicants are announced to the public in the subsequent month's post.
```

File: wiki/People/Project_Loved_Team/en.md [hunk 2]
```diff
@@ -6,14 +6,29 @@ tags:
...
+
+Some team additions may be added manually on a case-by-case basis in order to fill in positions that require certain skills or permissions. (e.g. the content moderator position, which is exclusive to the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) members)
```
Comment (Walavouchey) [inline wiki/People/Project_Loved_Team/en.md:25]:
```suggestion
Some team additions may be added manually on a case-by-case basis in order to fill in positions that require certain skills or permissions. (e.g. the content moderator position, which is exclusive to [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) members)
```

File: wiki/People/Project_Loved_Team/en.md [hunk 3]
```diff
@@ -6,14 +6,29 @@ tags:
...
+
 ## Trivia
 
 - ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689) created the original team of captains in September 2017 along with the [Captain's Pick](/wiki/History_of_osu!/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) system that was later renamed to Project Loved. Until late 2019, he oversaw and managed the whole project.
 - Prior to the 28th of April, 2021, the [Project Loved group page](https://osu.ppy.sh/groups/31) was not public and used only for administrative purposes. As of now, it is publicly available.
   - Until the 16th of April, 2021, the only people that had been in the group were ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350), ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237), ::{ flag=SG }:: [Shoegazer](https://osu.ppy.sh/users/2520707), and ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689). Now, all of the Project Loved Team belongs to the group.
+- As of April 28th, 2023, team members are awarded badges based on the length of their tenure in the team. The badges range from 1 year to 5 years.
```
Comment (Walavouchey) [inline wiki/People/Project_Loved_Team/en.md:34]:
```suggestion
- As of 28 April 2023, team members are awarded badges based on the length of their tenure in the team. The badges range from 1 year to 5 years.
```

File: wiki/People/Project_Loved_Team/en.md [hunk 4]
```diff
@@ -6,14 +6,29 @@ tags:
...
+
+The criteria for joining the team vastly differs from role to role, but proficiency and community presence is always sought after.
+
 ## Trivia
 
 - ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689) created the original team of captains in September 2017 along with the [Captain's Pick](/wiki/History_of_osu!/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) system that was later renamed to Project Loved. Until late 2019, he oversaw and managed the whole project.
 - Prior to the 28th of April, 2021, the [Project Loved group page](https://osu.ppy.sh/groups/31) was not public and used only for administrative purposes. As of now, it is publicly available.
   - Until the 16th of April, 2021, the only people that had been in the group were ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350), ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237), ::{ flag=SG }:: [Shoegazer](https://osu.ppy.sh/users/2520707), and ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689). Now, all of the Project Loved Team belongs to the group.
```
Comment (Walavouchey) [inline wiki/People/Project_Loved_Team/en.md:33]:
```suggestion
- Prior to the 28 April 2021, the [Project Loved group page](https://osu.ppy.sh/groups/31) was not public and used only for administrative purposes. As of now, it is publicly available.
  - Until the 16 April 2021, the only people that had been in the group were ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350), ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237), ::{ flag=SG }:: [Shoegazer](https://osu.ppy.sh/users/2520707), and ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689). Now, all of the Project Loved Team belongs to the group.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Community/Project_Loved/en.md [hunk 5]
```diff
@@ -79,8 +79,8 @@ For any questions regarding what goes in each category, contact the osu! Loved c
...
 - **The beatmap must have at least 30 favourites.** This is a minimum threshold to indicate a positive and existing community reaction to the beatmap. Brigading for favourites is discouraged, although captains cannot verify whether that happens.
-- **Every person involved in the beatmap must not disapprove of it being Loved.** This includes guest difficulty creators, hitsounders, skinners, and storyboarders. Users who are inactive for over a year, have not expressed disapproval of the nomination of the map, and are unreachable through osu! private messages do not need to be contacted.
-- **The beatmap creator cannot be banned or restricted.** Guest difficulties from restricted users will be judged on a case-by-case basis once the map is considered for nomination by a Loved captain.
+- **Every person involved in the beatmap must not disapprove of it being Loved.** This includes guest difficulty creators, hitsounders, skinners, and storyboarders. If one of the users involved in the mapset explicitly states their disagreement with their contribution being Loved, then it is to be removed from the mapset after its nomination.
```
[the remark this answers is not in the document]
Comment (Hiviexd) [reply wiki/Community/Project_Loved/en.md:82]: i went with the vague route to include other contributions like sb, hs, etc

[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Community/Project_Loved/en.md:82]: hmm, ok then

```suggestion
- **Every person involved in the beatmap must not disapprove of it being Loved.** This includes guest difficulty creators, hitsounders, skinners, and storyboarders. If any user involved in the beatmap explicitly disagrees with their contribution being Loved, then it is to be removed from the beatmap after its nomination.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**